python/ray/tune/sync_client.py

```diff
@@ -214,6 +219,10 @@ def sync_down(self, source, target):
         return self._execute(self.sync_down_template, source, target)
 
     def delete(self, target):
+        if self._closed:
+            logger.warning(
+                "Sync client is already closed. Skipping subsequent calls.")
```

Comment:
I'm interested in understanding when this occurs. Can we make sure that this isn't called if closed?